projects/packages/waf/src/class-brute-force-protection.php

```diff
@@ -498,16 +503,9 @@ public function log_failed_attempt( $login_user = null ) {
 		 * @param array Information about failed login attempt
 		 *   [
 		 *     'login'             => (string) Username or email used in failed login attempt
-		 *     'has_login_ability' => (bool) Whether the user has the ability to login based on their IP address
 		 *   ]
 		 */
-		do_action(
-			'jpp_log_failed_attempt',
-			array(
-				'login'             => $login_user,
-				'has_login_ability' => $this->has_login_ability(),
-			)
-		);
+		do_action( 'jpp_log_failed_attempt', array( 'login' => $login_user ) );
```

Comment:
Since we're changing the structure of the passed argument, I have two questions:

- Should we describe the change in the comment section?
- Is there any other code somewhere that relies on the `has_login_ability` key being present? If so, it will probably emit a warning. Maybe we should avoid changing this code for now?